### ISSUE TITLE
"setup.py" or "setup.cfg" error code while installing dependencies

### ISSUE THREAD

**Issue by Xayzer2** [carson-katri/dream-textures#47]
Full Error message:

ERROR: File "setup.py" or "setup.cfg" not found. Directory cannot be installed in editable mode: C:\Users\Xayzer\AppData\Roaming\Blender Foundation\Blender\3.3\scripts\addons\dream-textures-main\stable_diffusion

I downloaded the zip from the releases tab, started blender as administrator, and I used regedit to set LongPathsEnabled to 1.

**Comment by carson-katri** [comment]
It looks like you downloaded the wrong zip, you need to download the one called `dream_textures.zip` not the source code.

**Comment by Xayzer2** [comment]
Thank you very much for the reply! Made me realize I had made a very stupid mistake. I had actually downloaded the correct zip file (dream_textures.zip), however a few days ago, I had also downloaded the file from Code/Download ZIP, and since that was in the default directory where I store my downloaded Blender addons, I kept installing that one instead of the correct zip, which ended up in another directory. As I said, very stupid mistake.
